src/GameFace/Modules/ModuleTasks.cs

```diff
@@ -0,0 +1,78 @@
+﻿using Nancy;
+using Nancy.ModelBinding;
+using GameFace.Controllers;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Collections.Generic;
+using GameFace.Models;
+using Microsoft.EntityFrameworkCore;
+using System;
+
+namespace GameFace.Modules
+{
+    public class ModuleTasks : Nancy.NancyModule
+    {
+
+        public ModuleTasks() : base("tasks")
+        {
+
+
+            Post("/", async args =>
+            {
+                var creationAttempt = this.Bind<Tasks>(c => c.id);
```

Comment:
We will need to do some validation - if the frequency is 0, it will blow up. We might also care about if a name was supplied if desirability is positive etc - I will leave the details up to you.